docs/configuration.md

```diff
@@ -1377,6 +1377,91 @@ It will also use different parameters for test suites.
 
 > This parameter does not modify which files are formatted.
 
+## Literals
+
+> Since v2.5.0.
+
+Scalafmt allows flexible configuration of [Integer](https://www.scala-lang.org/files/archive/spec/2.13/01-lexical-syntax.html#integer-literals)
+and [Floating Point](https://www.scala-lang.org/files/archive/spec/2.13/01-lexical-syntax.html#integer-literals) literals formatting.
+
+Default formatting:
+
+```scala mdoc:scalafmt
+---
+123l
+0XFff
+0x1Abl
+10E-1
+10e-1D
+```
+
+Each `literals.*` setting has three available options:
+`Upper`, `Lower`, `Unchanged`.
+
+### `literals.long`
+
+Responsible for the case of `Long` literals suffix `L`
+
+```scala mdoc:scalafmt
+literals.long=Upper
+---
+123l
+```
+
+### `literals.float`
```

Comment:
do we need to add the default section for each of these? or is `Unchanged` default everywhere (in which case we should mention that once, at the top)?

Reply:
Yes, defaults will not be superfluous